shukla/README.md

```diff
@@ -0,0 +1,22 @@
+1. Forked repository, cloned onto local device
+
+2. Download VSCode for Ubuntu Virtual Machine.
+
+3. Created new branch "ukshukla" and created directory "shukla", with this README.md file in it.
+
+4. Created basic helloWorld.java program, which prints "Hello world!". added Makefile to build and run this program. To run, type in "make run" in the terminal. Afterwards, make sure to clean out all the .class files by running "make clean".
+
+5. Installed pre-commit. Created basic config file [.pre-commit-config.yaml] with auto-generated contents, with the "pre-commit sample-config" command.
+
+6. Installed git hook scripts, and ran hooks against all files.
+
+7. Worked through the Docker Carpentries module.
+
+8. Created Dockerfile which runs helloWorld.java.
+
+9. To build this Dockerfile, execute the following build commands:
+	A) docker build -t urmishukla/ubuntu-java:v1 .
+	B) run "docker image ls" to check that this new Docker image is present.
+	C) run "docker run urmishukla/ubuntu-java:v1"
```

Comment:
```suggestion
	C) run "docker run --rm urmishukla/ubuntu-java:v1"
```

If `docker run` is executed without the `--rm` flag then each time it is run it will create a container and leave it until another process cleans it up explicitly. While there are advantages to this if you might want to reuse an existing container instance in the future for desktop use you don't want to create lots of copies that just pile up until you explicitly remove them with `docker stop` and `docker rm`. You can see all the containers that exist that haven't been removed with `docker ps -a`.